packages/forms/src/elements/Range.js

```diff
@@ -34,33 +33,25 @@ const Range = React.forwardRef(({ min, max, step, ...otherProps }, ref) => {
 
       setBackgroundSize(`${percentage}%`);
     },
-    [max, min]
+    /* eslint-disable-next-line react-hooks/exhaustive-deps */
+    [max, min, step]
```

Comment:
Could you clarify the purpose of adding `step` into the dependency array?

I pulled down the branch locally and remove `step` from the dependency array and did not notice a difference with `<Range>`. Does the `react-hooks/exhaustive-deps` lint rule not apply in this particular situation? I think that `step` is not needed in the dependency array since `step` isn't used within the `useCallback` code — but maybe there is something I'm missing?

Reply:
with `step` removed from the array:
- use the "knobs" to set the step value to 1
- slide thumb to a value
- change knob to 50
result: background size fails to update.

Reply:
Awesome, this makes sense. The `step` value is an implicit dependency.